src/components/pages/about-us/our-team/our-team.tsx

```diff
@@ -1,4 +1,4 @@
-import Image from 'next/image';
+of Digital Communication & Marketingimport Image from 'next/image';
```

Comment:
Missing carriage return, at least,  don't you think?

Reply:
indeed. The correct one would start with "Head"